tests/misc/test_uefi_boot/README.rst

```diff
@@ -0,0 +1,50 @@
+UEFI Boot
+###########
```

Comment:
I'd put this documentation in the platform layer somewhere.  This test isn't the only place where one might want to build EFI code under qemu.